clients/admin-ui/src/features/system/add-multiple-systems/AddMultipleSystems.tsx

```diff
@@ -274,20 +273,6 @@ export const AddMultipleSystems = ({ redirectRoute }: Props) => {
     .getSelectedRowModel()
     .rows.some((row) => !row.original.linked_system);
 
-  const isTooltipDisabled = useMemo(() => {
-    /*
-      The tooltip surrounding the add button is conditionally displayed.
-
-      It displays if no rows have been selected or if all of the vendors
-      are already linked to systems
-    */
-    if (!anyNewSelectedRows || allRowsLinkedToSystem) {
-      return false;
-    }
-
-    return true;
-  }, [anyNewSelectedRows, allRowsLinkedToSystem]);
-
```

Comment:
after some investigation, I learned that this state is impossible given the current code conditions, so I removed it altogether.